Cargo.toml

```diff
@@ -13,7 +13,7 @@ figment = { version = "0.10.8", features = ["toml", "env"] }
 flexi_logger = "0.24.2"
 log = "0.4.17"
 rs-cnc = { git = "https://github.com/astriaorg/rs-cnc", rev = "e0b45d6" }
-sequencer-relayer = { git = "https://github.com/astriaorg/sequencer-relayer", rev = "6c6cb95" }
+sequencer-relayer = { git = "https://github.com/astriaorg/sequencer-relayer" }
```

Comment:
Using `rev` was a suggestion from Janis. I do like the idea of pinning versions, but can see the annoyance while we're developing the PoC. I guess ideally we would have this in an actual crate and have no need for `rev`. Is that the thinking here? Just as a shortcut to ease development till we have finalized our crate usage strategy?

Reply:
ah I just removed the rev since we're on sequencer relayer master now, but yeah we should probably pin it to that commit still!